modules/distribution/pom.xml

```diff
@@ -260,6 +260,7 @@
                                 <replace file="target/wso2carbon-core-${carbon.kernel.version}/repository/conf/carbon.xml" token="${product.version}" value="${project.version}" />
                                 <replace file="../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/resources/conf/templates/repository/conf/carbon.xml.j2" token="${product.version}" value="{{product.version}}" />
                                 <replace file="../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/resources/conf/default.json" token="${product.version}" value="${project.version}" />
+                                <replace file="../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/resources/conf/default.json" token="&quot;authorization_manager.properties.GroupAndRoleSeparationEnabled&quot;: &quot;false&quot;" value="&quot;authorization_manager.properties.GroupAndRoleSeparationEnabled&quot;: &quot;true&quot;" />
```

Comment:
can't we do this at our default.json?

Reply:
also, we need to follow the toml convention for this key. Would need to change this or add a mapping so that it would be,
`group_and_role_separation_enabled`

Reply:
We have changed the default.json value in the product level as this config comes from kernel.

Reply:
Added an issue to track the config name change: https://github.com/wso2/product-is/issues/9641